.github/workflows/dependency-review.yaml

```diff
@@ -6,7 +6,7 @@ on:
 
 jobs:
   dependency-review:
-    uses: gravitational/shared-workflows/.github/workflows/dependency-review.yaml@main
+    uses: gravitational/shared-workflows/.github/workflows/dependency-review.yaml@avatus/add_more_deps
```

Comment:
Could we refer this to a "proper" branch or a tag?

Reply:
You caught me in the middle of testing a `shared-workflows` change. I am updating back to main, but only wanted to make sure my change actually got the workflow to pass.